packages/components/src/templates/next/components/internal/BaseParagraph/BaseParagraph.tsx

```diff
@@ -1,15 +1,17 @@
 import type { Node } from "interweave"
 import { Interweave } from "interweave"
-import { polyfill } from "interweave-ssr"
 
 import type { BaseParagraphProps } from "~/interfaces/native/Paragraph"
 import { twMerge } from "~/lib/twMerge"
 import { getReferenceLinkHref, isExternalUrl } from "~/utils"
 import { Link } from "../Link"
 
-// NOTE: We need this polyfill as interweave uses a DOM to perform the
-// conversion of HTML to React components
-polyfill()
+// This will be tree-shaken out of client bundles
+if (typeof window === "undefined") {
+  void import("interweave-ssr").then(({ polyfill }) => {
+    polyfill()
+  })
```

Comment:
nit: can retain the original comment on why we need this polyfill

Reply:
@dcshzj done! https://github.com/opengovsg/isomer/pull/1460/commits/737026483e20f1ed2bd8db9375e80372beccaf50